src/main/java/com/sprint/mission/discodeit/repository/file/AbstractFileRepository.java

```diff
@@ -1,11 +1,13 @@
 package com.sprint.mission.discodeit.repository.file;
 
 import com.sprint.mission.discodeit.entity.BaseEntity;
+import lombok.extern.slf4j.Slf4j;
 
 import java.io.*;
 import java.util.*;
 import java.util.concurrent.ConcurrentHashMap;
 
+@Slf4j
 public class AbstractFileRepository<T extends BaseEntity> {
     protected final Map<UUID, T> data;
     private final String FILE_PATH;
```

Comment:
```suggestion
    private final String filePath;
```
상수인 경우에 FILE_PATH를 사용합니다. 하지만 일반 필드의 경우 filePath와 같이 카멜케이스를 사용해요.